modules/administration-guide/pages/the-woopra-telemetry-plugin.adoc

```diff
@@ -8,22 +8,29 @@
 
 = The Woopra telemetry plug-in
 
-The link:https://github.com/che-incubator/che-workspace-telemetry-woopra-plugin[Woopra Telemetry Plugin] is a plug-in built to send telemetry from a {prod} installation to Segment and Woopra.
+The link:https://github.com/che-incubator/devworkspace-telemetry-woopra-plugin[Woopra Telemetry Plugin] is a plug-in built to send telemetry from a {prod} installation to Segment and Woopra.
 This plug-in is used by link:https://workspaces.openshift.com[Eclipse Che hosted by Red Hat], but any {prod} deployment can take advantage of this plug-in.
 There are no dependencies other than a valid Woopra domain and Segment Write key.
-The link:https://raw.githubusercontent.com/che-incubator/che-workspace-telemetry-woopra-plugin/master/meta.yaml[plug-in's meta.yaml] file has 5 environment variables that can be passed to the plug-in:
+The plug-in's  link:https://raw.githubusercontent.com/che-incubator/devworkspace-telemetry-woopra-plugin/main/plugin.yaml[plugin.yaml] devfile v2 file has 4 environment variables that can be passed to the plug-in:
```

Comment:
The build fails because this file: https://raw.githubusercontent.com/che-incubator/devworkspace-telemetry-woopra-plugin/main/plugin.yaml

does not exist yet.

It will however, when this PR https://github.com/che-incubator/devworkspace-telemetry-woopra-plugin/pull/9 is merged.

Reply:
https://github.com/che-incubator/devworkspace-telemetry-woopra-plugin/pull/9 has been megred